pkg/k8s/apis/cilium.io/client/crds/v2/ciliumnodes.yaml

```
@@ -215,6 +215,15 @@ spec:
                     description: IPv6 is the IPv6 address of the IPv4 health endpoint.
                     type: string
                 type: object
+              ingress:
```

If we're extending the CRD type here to add a new field, I'm sure we'll bump the CRD version at some point here, either in this PR or immediately prior to the next (RC) release. What I'm wondering though is what happens if a user upgrades to this version, then downgrades againto the current version (eg 1.11.5). The older version will not really understand the new CRD version, so will it just silently ignore this new (optional) field?

---

Thanks for the heads up on the version bump, forgot about that. I recall unknown fields are ignored.